docs/search_dsl.rst

```diff
@@ -414,6 +413,30 @@ endpoint) you can do so via ``execute_suggest``:
 
     print(suggestions.my_suggestion)
 
+
+More Like This Query
+~~~~~~~~~~~~~~~~~~~~
+
+To use Elasticsearch's more_like_this functionality, you can use the MoreLikeThis query type.
+
+A simple example is below
+
+.. code:: python
+    from elasticsearch_dsl.query import MoreLikeThis
+    from elasticsearch_dsl Search
+
+    my_text = 'I want to find something similar'
+
+    s = Search()
+    # We're going to match based only on two fields, in this case text and title
+    s = s.query(MoreLikeThis(like=my_text, fields=['text', 'title]))
+    # You can also exclude fields from the result to make the response quicker in the normal way
+    s = s.source(exclude=["text"])
+    response = s.execute().to_dict()
```

Comment:
Could we avoid working with the raw `dict`s here? I understand it is sometimes needed but we should document the full API which includes the `Hit` wrapper, maybe just add a tiny `for` loop that will just print out the `title` fields of found documents?
```python
for hit in response:
    print(hit.title)
```